pkg/clients/openshift/metadata.go

```diff
@@ -0,0 +1,59 @@
+package openshift
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	metadataConfigMap = "osd-cluster-metadata"
+	configNamespace   = "openshift-config"
+)
+
+// getOsdClusterMetadata returns osd-cluster-metadata configmap data array from openshift-config namespace
+// this contains metadata about the cluster
+func (c Client) getOsdClusterMetadata(ctx context.Context) (map[string]string, error) {
+	var cm corev1.ConfigMap
+	cmdata := new(map[string]string)
+	if err := c.Get(ctx, metadataConfigMap, configNamespace, &cm); err != nil {
+		return *cmdata, err
+	}
+	return cm.Data, nil
+}
+
+func (c Client) IsSts(ctx context.Context) (bool, error) {
+	var cmData map[string]string
+	var err error
+	if cmData, err = c.getOsdClusterMetadata(ctx); err != nil {
+		return false, err
+	}
+	return cmData["api_openshift_com_sts"] == "true", nil
+}
+
+func (c Client) IsCcs(ctx context.Context) (bool, error) {
```

Comment:
```suggestion
func (c Client) IsCCS(ctx context.Context) (bool, error) {
```

let's capitalize acronyms (STS)